menu.js

```diff
@@ -13,4 +15,379 @@ function closeMenu() {
+projectDeskTagsList3.innerHTML = projectData[0].desktopTechnologies.data6;
+projectDeskTags.appendChild(projectDeskTagsList3);
+const projectDeskDesc = document.createElement('p');
+projectDeskDesc.classList.add('descriptionParagraph');
+projectDeskDesc.innerHTML = projectData[0].description;
+projectDeskDiv.appendChild(projectDeskDesc);
+const closeButton = document.getElementsByClassName('closeBtn');
+
+function closeDeskpop() {
+  workCard.removeChild(projectDeskDiv);
+  overlay.classList.remove('active');
+}
+
+function displayDesktopCard() {
+  workCard.appendChild(projectDeskDiv);
+  closeButton[0].addEventListener('click', closeDeskpop);
+  overlay.classList.add('active');
+}
+
+projectDetails.forEach((n) => n.addEventListener('click', displayDesktopCard));
+
+// ..........................................
+
+const projectDiv = document.createElement('div');
+projectDiv.classList.add('mcardContainer');
+const projectSpan = document.createElement('span');
+projectSpan.classList.add('mcloseBtn');
+projectSpan.innerHTML = projectData[0].popCancelBtn;
+projectDiv.appendChild(projectSpan);
+const projectImg = document.createElement('img');
+projectImg.classList.add('mProjectImg');
+projectImg.src = projectData[0].desktopPopImage;
+projectDiv.appendChild(projectImg);
+const projectHeader = document.createElement('h3');
+projectHeader.classList.add('mpopPrimaryText');
+projectHeader.innerHTML = projectData[0].popHeaderDescription;
+projectDiv.appendChild(projectHeader);
+const projectTags = document.createElement('ul');
+projectTags.classList.add('mTags');
+projectDiv.appendChild(projectTags);
+const projectTagsList1 = document.createElement('li');
+projectTagsList1.classList.add('mTagsList');
+projectTagsList1.innerHTML = projectData[0].mobileTechnologies.data1;
+projectTags.appendChild(projectTagsList1);
+const projectTagsList2 = document.createElement('li');
+projectTagsList2.classList.add('mTagsList');
+projectTagsList2.innerHTML = projectData[0].mobileTechnologies.data2;
+projectTags.appendChild(projectTagsList2);
+const projectTagsList3 = document.createElement('li');
+projectTagsList3.classList.add('mTagsList');
+projectTagsList3.innerHTML = projectData[0].mobileTechnologies.data3;
+projectTags.appendChild(projectTagsList3);
+const projectDesc = document.createElement('p');
+projectDesc.classList.add('mdescriptionParagraph');
+projectDesc.innerHTML = projectData[0].description;
+projectDiv.appendChild(projectDesc);
+const buttonSeeLive = document.createElement('button');
+buttonSeeLive.classList.add('mseelive');
+buttonSeeLive.innerHTML = projectData[0].seeLive;
+projectDiv.appendChild(buttonSeeLive);
+const buttonSeeSource = document.createElement('button');
+buttonSeeSource.classList.add('mseesource');
+buttonSeeSource.innerHTML = projectData[0].seeSource;
+projectDiv.appendChild(buttonSeeSource);
+
+function closeMobilepop() {
+  mworkCard.removeChild(projectDiv);
+  overlay.classList.remove('active');
+}
```

Comment:
- [ ] Your close buttons are working but they have an issue that must be resolved: when you close it on mobile it's closing but it's still showing at the desktrop sizes. Kindly fix it to make it more functional.
https://app.flonnect.com/view/video/saranlitugsem/63812d20e746641a4b9fd658